File: trabajo1-poo/src/gestorAplicacion/school_related/EvaluationEvent.java
package gestorAplicacion.school_related;

import java.io.Serializable;

/*
 * Una evaluacion con su nota individual.
 * Cada materia tiene 3 evaluaciones, las notas son, por simplicidad, aleatorias para cada asignatura para 
 * cada estudiante. Se usan para calcular el rendimiento de un curso, profesor o estudiante.
 * 
 */

public class EvaluationEvent implements Serializable {
	
	private static final long serialVersionUID = 7774322636999465955L;
	private final String subject_name;
	
	private final String name;
	private double grade;
	
	public EvaluationEvent(String subname, String evname)
	{
		this.subject_name = subname;
		this.name = evname;	
	}
	
	public void setGrade(float grade)
	{
		this.grade = grade;
	}

	public String getSubject_name() {
		return subject_name;
	}

	public String getName() {
		return name;
	}
	
	public double getGrade()
	{
		return this.grade;
	}

}

File: Python/logical_impl/EvaluationEvent.py
class EvaluationEvent:
    def __init__(self, grade, name, sn):
        self.grade = grade
        self.evname = name
        self.subject_name = sn

    def getGrade(self):
        return self.grade

    def setGrade(self, g):
        self.grade = g

    def getEvname(self):
        return self.evname

    def setEvname(self, g):
        self.evname = g

    def getSubject_name(self):
        return self.subject_name

    def setSubject_name(self, g):
        self.subject_name = g

